src/js/components/DataTable/DataTable.js

```diff
@@ -127,10 +127,22 @@ const DataTable = ({
 
   // placeholder placement stuff
   const headerRef = useRef();
+  const bodyRef = useRef();
   const footerRef = useRef();
   const [headerHeight, setHeaderHeight] = useState();
   const [footerHeight, setFooterHeight] = useState();
 
+  const [overflowComp, setOverflowComp] = useState(0);
```

Comment:
would it make sense to call this state `bodyOverflow` vs `overflowComp`?

Reply:
I was going for `overflowCompensation` but I do agree it could use some work on readability. I like `bodyOverflow` but I'm not sure if it's related to what it does. Even though `body` is the one overflowing, I don't think it would explain its use.

WDYT about `scrollBarWidth` or maybe `scrollBarCompensation` ?

Also, I'll try pushing at least one snapshot later on, so we can keep track of this behavior.

Reply:
We've used the term `offset` for situations like this elsewhere. Perhaps `scrollOffset`?